src/earn/PoolCard.tsx

```diff
@@ -96,20 +99,40 @@ export default function PoolCard({
     <Touchable borderRadius={12} style={styles.card} testID={testID} onPress={onPress}>
       <View style={styles.cardView}>
         <View style={styles.titleRow}>
-          {tokensInfo.map((token, index) => (
-            <TokenIcon
-              key={index}
-              token={token}
-              viewStyle={index > 0 ? { marginLeft: -8, zIndex: -index } : {}}
-            />
-          ))}
-          <View style={styles.titleTextContainer}>
-            <Text style={styles.titleTokens}>
-              {tokensInfo.map((token) => token.symbol).join(' / ')}
-            </Text>
-            <Text style={styles.titleNetwork}>
-              {t('earnFlow.poolCard.onNetwork', { networkName: NETWORK_NAMES[networkId] })}
-            </Text>
+          <View style={styles.titleRowToken}>
+            {tokensInfo.map((token, index) => (
+              <TokenIcon
+                key={index}
+                token={token}
+                viewStyle={index > 0 ? { marginLeft: -8, zIndex: -index } : {}}
+              />
+            ))}
+            <View style={styles.titleTextContainer}>
+              <Text style={styles.titleTokens}>
+                {tokensInfo.map((token) => token.symbol).join(' / ')}
+              </Text>
+              <Text style={styles.titleNetwork}>
+                {t('earnFlow.poolCard.onNetwork', { networkName: NETWORK_NAMES[networkId] })}
+              </Text>
+            </View>
+          </View>
+          <View>
+            <Text style={styles.safetyText}>{t('earnFlow.poolCard.safety')}</Text>
+            {safety && (
+              <View style={styles.tripleBarContainer}>
+                {BAR_HEIGHTS.map((height, index) => (
```

Comment:
could be in a follow up issue, but would be nice if this was a shared component with the SafetyCard